### DISCUSSION TITLE
Search for vulnerabilities by dependency class

### DISCUSSION THREAD

**Discussion by alexr3** [DefectDojo/django-DefectDojo#7436] [Q&A] [answered]
Hello everyone!

Is it possible to search for vulnerabilities by dependency class (to find a vulnerability in a parent or child class)?

**Comment by manuel-sommer** [comment, marked as answer]
For that, I recommend DependencyTrack. You can use DependencyTrack very good together with DefectDojo and sync the findings over.

**Reply by manuel-sommer** [reply]
Can you close this Q&A @mtesauro and mark this as answered?
Just going through various open discussions and looking where I can help to clear out open topics.
